plugins/woocommerce/src/Internal/Admin/Loader.php

```diff
@@ -9,6 +9,8 @@
 use Automattic\WooCommerce\Admin\Features\Features;
 use Automattic\WooCommerce\Admin\PageController;
 use Automattic\WooCommerce\Admin\PluginsHelper;
+use Automattic\WooCommerce\Internal\Admin\ProductReviews\Reviews;
```

Comment:
This class has a few GH action failure messages, but apparently not related to this class' changes. Just noting, but I don't think we should fix them here unless directly related to the changes introduced.

Reply:
I agree we shouldn't fix the code sniff issues in the Loader, as they are not introduced by this PR.